LICENCE

```diff
@@ -1,190 +0,0 @@
-                                 Apache License
```

Comment:
@inalgnu can you confirm that we can safely remove this LICENCE?

Reply:
yes! 
My bad, it was a wrong file name